scout/ecm_prep.py

```diff
@@ -6575,7 +6581,7 @@ def gen_tsv_facts(
 
     def apply_tsv(self, load_fact, ash_cz_wts, eplus_bldg_wts,
                   cost_fact_hourly, carbon_fact_hourly, mskeys, bldg_sect,
-                  eu, opts, cost_yr_map, carb_yr_map):
+                  eu, opts, cost_yr_map, carb_yr_map, save_shp_warn):
```

Comment:
I think it would make sense to initialize `save_shp_warn` in this method instead of having as an argument, as it does not appear to be returned or updated anywhere outside of `apply_tsv`.

Initializing as an empty list would work before ` for bldg in eplus_bldg_wts.keys():`

Reply:
That list stores previous warnings generated within `apply_tsv` to avoid duplication of warning messages. If it's re-initialized each time `apply_tsv` is run, that history of warning messages would be removed.